pkg/analyzer/nodes/node.go

```diff
@@ -89,7 +89,7 @@ func extractFilesFromDirectory(root string, matcher matcher.PrefixMatcher) (*fil
 		files: make(map[string]*fileMetadata),
 	}
 	m := metrics.FileExtractionMetrics{}
-	for _, dir := range matcher.GetCommonPrefixDirs() {
+	for _, dir := range []string{"etc/", "usr/share/rpm", "var/lib/rpm"} { //TODO(ROX-13771): Use range matcher.GetCommonPrefixDirs() again after fixing
```

Comment:
To make the style-check happy:

```suggestion
        // TODO(ROX-13771): Use range matcher.GetCommonPrefixDirs() again after fixing
	for _, dir := range []string{"etc/", "usr/share/rpm", "var/lib/rpm"} { 
```